### ISSUE TITLE
Icon cannot be found error thrown in PHPStorm 2018.2 EAP

### ISSUE THREAD

**Issue by Synchro** [psliwa/idea-composer-plugin#61]
```
java.lang.Throwable: Icon cannot be found in 'icons/packagist.png', aClass='class org.psliwa.idea.composerJson.Icons$'
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:123)
	at com.intellij.openapi.util.IconLoader.getIcon(IconLoader.java:148)
	at com.intellij.openapi.util.IconLoader.getIcon(IconLoader.java:109)
	at org.psliwa.idea.composerJson.Icons$.<init>(Icons.scala:9)
	at org.psliwa.idea.composerJson.Icons$.<clinit>(Icons.scala)
	at org.psliwa.idea.composerJson.intellij.filetype.ComposerJsonFileType.getIcon(ComposerJsonFileType.scala:14)
	at com.intellij.psi.impl.ElementBase.getBaseIcon(ElementBase.java:132)
	at com.intellij.psi.impl.ElementBase.computeBaseIcon(ElementBase.java:115)
	at com.intellij.psi.impl.ElementBase.lambda$computeIcon$1(ElementBase.java:93)
	at com.intellij.util.AstLoadingFilter.disableTreeLoading(AstLoadingFilter.java:126)
	at com.intellij.util.AstLoadingFilter.disableTreeLoading(AstLoadingFilter.java:115)
	at com.intellij.psi.impl.ElementBase.computeIcon(ElementBase.java:93)
	at com.intellij.psi.impl.ElementBase.getIcon(ElementBase.java:74)
	at com.intellij.ide.projectView.impl.nodes.AbstractPsiBasedNode.a(AbstractPsiBasedNode.java:137)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:915)
	at com.intellij.ide.projectView.impl.nodes.AbstractPsiBasedNode.update(AbstractPsiBasedNode.java:126)
	at com.intellij.ide.util.treeView.PresentableNodeDescriptor.getUpdatedPresentation(PresentableNodeDescriptor.java:88)
	at com.intellij.ide.util.treeView.PresentableNodeDescriptor.update(PresentableNodeDescriptor.java:41)
	at com.intellij.ui.tree.StructureTreeModel$Node.f(StructureTreeModel.java:280)
	at com.intellij.ui.tree.StructureTreeModel$Node.<init>(StructureTreeModel.java:262)
	at com.intellij.ui.tree.StructureTreeModel$Node.<init>(StructureTreeModel.java:256)
	at com.intellij.ui.tree.StructureTreeModel.c(StructureTreeModel.java:235)
	at com.intellij.ui.tree.StructureTreeModel.a(StructureTreeModel.java:139)
	at com.intellij.ui.tree.StructureTreeModel.isLeaf(StructureTreeModel.java:175)
	at com.intellij.ui.tree.AsyncTreeModel$CmdGetChildren.a(AsyncTreeModel.java:575)
	at com.intellij.ui.tree.AsyncTreeModel$CmdGetChildren.getNode(AsyncTreeModel.java:550)
	at com.intellij.ui.tree.AsyncTreeModel$ObsolescentCommand.get(AsyncTreeModel.java:442)
	at com.intellij.ui.tree.AsyncTreeModel$ObsolescentCommand.get(AsyncTreeModel.java:408)
	at com.intellij.util.concurrency.Command$Processor.a(Command.java:48)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1151)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.a(ProgressIndicatorUtils.java:84)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.a(ProgressIndicatorUtils.java:137)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:580)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:525)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:85)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:144)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:126)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:84)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:105)
	at com.intellij.util.concurrency.Invoker.invokeSafely(Invoker.java:114)
	at com.intellij.util.concurrency.Invoker.a(Invoker.java:76)
	at com.intellij.util.concurrency.Invoker$BackgroundThread.b(Invoker.java:248)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2$1.run(BoundedTaskExecutor.java:212)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:229)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:205)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

PhpStorm 2018.2 EAP
Build #PS-182.3208.33, built on June 13, 2018
PhpStorm EAP User
Expiration date: July 13, 2018
JRE: 1.8.0_152-release-1226-b7 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.11.6

This error seems to prevent the project panel from loading.

**Comment by pxgamer** [comment]
Just to flag up, this has also been reported as an issue on YouTrack. 🙂
https://youtrack.jetbrains.com/issue/WI-42534

**Comment by Synchro** [comment]
Thanks. I've also confirmed that disabling the plugin allows the project pane to work again.

**Comment by pxgamer** [comment]
Apparently this should be resolved by JetBrains/intellij-community@d78491486cd1d6013cd41901f08f6f3eff8a39c8 👍

**Comment by krasa** [comment]
Nope, but they will have to fix it  :-)

**Comment by codemasher** [comment]
just on a side note: it also breaks browsing files on a remot host.

**Comment by psliwa** [comment]
It was upstream bug introduced in the first 2018.2 EAP version. The newest EAP released today (182.3341.34) is fixed. Today I will include EAP tests in travis pipeline. If the issue persists, let me know, thanks :)
